core/arch/arm/plat-ls/conf.mk

```diff
@@ -98,6 +98,8 @@ $(call force,CFG_PL011,y)
 $(call force,CFG_CORE_ARM64_PA_BITS,48)
 $(call force,CFG_EMBED_DT,y)
 $(call force,CFG_EMBED_DTB_SOURCE_FILE,fsl-lx2160a-qds.dts)
+$(call force,CFG_LS_I2C,y)
+CFG_LS_I2C_TEST ?= n
```

Comment:
Should ```CFG_LS_I2C_TEST ``` depends on ```CFG_LS_I2C ``` ?

Reply:
Yes, That should depend on CFG_LS_I2C, Will update